updating README.md file with newly added --config option (#97)
rackerlabs/configsnap: README.md

Usage: configsnap [options]

Record useful system state information, and compare to previous state if run

> with PHASE containing "post" or "rollback". An optional file,

Reviewer: ```suggestion
with PHASE containing "post" or "rollback". A default config file,
```

> /etc/configsnap/additional.conf, can be provided for extra files, directories

Reviewer: ```suggestion
/etc/configsnap/additional.conf, can be customised to include extra files, directories
```

                        and --phase
  --pre=PRE_SUFFIX      suffix for files captured at previous state, for
                        comparison
  -c CONFIG, --config=CONFIG
                        additional config file to use. Setting this will

>                         overwrite default.

Reviewer: ```suggestion
                        override default.
```
